waspc/src/Wasp/Generator/SdkGenerator/Common.hs

```diff
@@ -73,9 +73,6 @@ relDirToRelFileP path = fromJust $ SP.parseRelFileP $ removeTrailingSlash $ SP.f
 makeSdkImportPath :: Path Posix (Rel SdkRootDir) File' -> Path Posix (Rel s) File'
 makeSdkImportPath path = [reldirP|wasp|] </> path
 
-extCodeDirInSdkRootDir :: Path' (Rel SdkRootDir) Dir'
-extCodeDirInSdkRootDir = [reldir|ext-src|]
```

Comment:
We accidentally had two constants describing the same thing, so I removed this one.